devices/stm32f446.yaml

```diff
@@ -80,8 +84,11 @@ _include:
  - ../peripherals/rcc/rcc_v2.yaml
  - ../peripherals/rcc/rcc_v2_i2s.yaml
  - ../peripherals/rcc/rcc_v2_i2s_pll.yaml
+ - ../peripherals/rcc/rcc_v2_i2s_pllp.yaml
  - ../peripherals/rcc/rcc_v2_i2s_pllq.yaml
  - ../peripherals/rcc/rcc_v2_i2s_pllm.yaml
+ - ../peripherals/rcc/rcc_v2_sai_pllp.yaml
```

Comment:
Here and the F469 use `rcc_v2_sai_pllp` but it refers to `PLLSAICFGR` whereas these devices have `PLLSAI1CFGR` with a 1-bit `PLLSAI1P` field (either 0 for P=7 or 1 for P=17)

Reply:
I'm not sure what you are talking about here. I only see references to PLLSAI1CFGR in the stm32l4 series not in the stm32f4 series.
Both the F446 and F469 have a PLLSAICFGR with a 2 bits PLLSAIP field, both in the SVD files and in the respectives RM.
Am I missing something ?

Reply:
Whoops, you're quite right, I was looking at the L4 RM for that file and got confused.